common/scaffolding/interactive-content-en.html

```diff
@@ -0,0 +1,387 @@
+	<td>Curabitur ut leo nisi</td>
+</tr>
+</table>
+<pre>
+<code>&lt;table&gt;
+&lt;colgroup&gt;
+	&lt;col span="1" style="background-color:red"&gt;
+&lt;tr&gt;
+	&lt;td&gt;Lorem ipsum dolor sit amet&lt;/td&gt;
+	&lt;td&gt;Proin pellentesque sodales cursus&lt;/td&gt;
+&lt;/tr&gt;
+&lt;tr&gt;
+	&lt;td&gt;Nunc dictum felis at dignissim dictum&lt;/td&gt;
+	&lt;td&gt;Curabitur ut leo nisi&lt;/td&gt;
+&lt;/tr&gt;
+&lt;/table&gt;</code>
+</pre>
+
+<h3>Draw graphics with the <code>canvas</code> element</h3>
+
+<canvas id="myCanvas">
+Your browser does not support the canvas tag.
+</canvas>
+
+<script>
+	let canvas = document.getElementById("myCanvas");
+	let ctx = canvas.getContext("2d");
+	ctx.fillStyle = "#FF0000";
+	ctx.fillRect(0, 0, 80, 80);
+</script>
+
+<pre>
+<code>&lt;canvas id="myCanvas"&gt;
+Your browser does not support the canvas tag.
+&lt;/canvas&gt;
+
+&lt;script&gt;
+	let canvas = document.getElementById("myCanvas");
+	let ctx = canvas.getContext("2d");
+	ctx.fillStyle = "#FF0000";
+	ctx.fillRect(0, 0, 80, 80);
+&lt;/script&gt;
+</code>
+</pre>
+
+<h3>The <code>template</code> element</h3>
+
+<p>Click the button below to display the hidden content from the template element.</p>
+
+<button onclick="showContent()">Show hidden content</button>
+
+<div id="content"></div>
+
+<template>
+<h2>Hidden Image</h2>
+<img src="img/input-image-sample.jpg" width="300" height="200">
+</template>
+
+<script>
+	function showContent() {
+	let temp = document.getElementsByTagName("template")[0];
+	let clone = temp.content.cloneNode(true);
+	let content = document.getElementById("content");
+	content.appendChild(clone);
+	}
+</script>
+
+<pre>
+<code>&lt;button onclick="showContent()"&gt;
+Show hidden content
+&lt;/button&gt;
+
+&lt;template&gt;
+	&lt;h2&gt;Hidden Image&lt;/h2&gt;
+	&lt;img src="img/input-image-sample.jpg" width="300" height="200"&gt;
+&lt;/template&gt;
+
+&lt;script&gt;
+	function showContent() {
+	let temp = document.getElementsByTagName("template")[0];
+	let clone = temp.content.cloneNode(true);
+	let content = document.getElementById("content");
+	content.appendChild(clone);
+	}
+&lt;/script&gt;
+</code>
+</pre>
+
+<h3>The <code>slot</code> element</h3>
+
+<template id="slotTemplate">
+<h1><slot name="heading"></slot></h1>
+<dl>
+<dt><slot name="term-1"></slot></dt>
+<dd><slot name="description-1"></slot></dd>
+<dt><slot name="term-2"></slot></dt>
+<dd><slot name="description-2"></slot></dd>
+</dl>
+</template>
+
+<div class="slotDiv">
+<span slot="heading">Lorem ipsum</span>
+<span slot="term-1">Dolor sit amet</span>
+<span slot="description-1">Aliquam vel consectetur erat</span>
+<span slot="term-2">Morbi vitae odio orci</span>
+<span slot="description-2">Cras neque ligula</span>
+</div>
+
+<script>
+var dlTemplate = document.querySelector('#slotTemplate').content;
+var div = document.querySelector('.slotDiv');
+
+div.attachShadow({  mode: 'open' }).appendChild(
+dlTemplate.cloneNode(true));
+</script>
```

Comment:
Make it work behind a button. That way we will be able to evaluate the initial state of those element and after they have play a role in the HTML transformation.

Reply:
Fixed in commit "Addressed changes".